<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>571</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>50</y>
          <width>151</width>
          <height>461</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="title">
         <string>Collect data</string>
        </property>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Port</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>90</y>
           <width>61</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Baudrate</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>140</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Samples</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="samples">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>160</y>
           <width>101</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>190</y>
           <width>61</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Filename</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="filename">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>210</y>
           <width>101</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_openFile">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>380</y>
           <width>81</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Open</string>
         </property>
        </widget>
        <widget class="QLabel" name="path">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>320</y>
           <width>121</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Open file:</string>
         </property>
        </widget>
        <widget class="QComboBox" name="baudrate">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>110</y>
           <width>101</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QComboBox" name="port">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>101</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_path">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>270</y>
           <width>31</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="iconSize">
          <size>
           <width>18</width>
           <height>18</height>
          </size>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QLabel" name="path_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>270</y>
           <width>91</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Path:</string>
         </property>
        </widget>
       </widget>
       <widget class="PlotWidget" name="graphWidget" native="true">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>60</y>
          <width>561</width>
          <height>221</height>
         </rect>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_play">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>31</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="iconSize">
         <size>
          <width>18</width>
          <height>18</height>
         </size>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_stop">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>10</y>
          <width>31</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="iconSize">
         <size>
          <width>18</width>
          <height>18</height>
         </size>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_save">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>10</y>
          <width>31</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>290</y>
          <width>561</width>
          <height>221</height>
         </rect>
        </property>
        <property name="text">
         <string>PLotter</string>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>391</width>
          <height>591</height>
         </rect>
        </property>
        <property name="title">
         <string>Train</string>
        </property>
        <widget class="QLineEdit" name="size">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>30</y>
           <width>171</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>30</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Layers</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>70</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Activation</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_9">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>110</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Optimizer</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_10">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>150</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Loss</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_11">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>200</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Metrics</string>
         </property>
        </widget>
        <widget class="QComboBox" name="Activation">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>70</y>
           <width>171</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QComboBox" name="Optimizer">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>110</y>
           <width>171</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QComboBox" name="Loss">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>150</y>
           <width>171</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QComboBox" name="Metrics">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>200</y>
           <width>171</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_createModel">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>240</y>
           <width>91</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Create</string>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>290</y>
           <width>351</width>
           <height>241</height>
          </rect>
         </property>
         <property name="text">
          <string>Structure of model</string>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_train">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>550</y>
           <width>81</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Train</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>10</y>
          <width>361</width>
          <height>591</height>
         </rect>
        </property>
        <property name="title">
         <string>Evaluate</string>
        </property>
        <widget class="QPushButton" name="pushButton_5">
         <property name="geometry">
          <rect>
           <x>-150</x>
           <y>580</y>
           <width>81</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Train</string>
         </property>
        </widget>
        <widget class="PlotWidget" name="widget" native="true">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>361</width>
           <height>251</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>290</y>
           <width>361</width>
           <height>281</height>
          </rect>
         </property>
         <property name="text">
          <string>Confussion Matrix</string>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="Tab3">
       <attribute name="title">
        <string>Page</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>811</width>
          <height>591</height>
         </rect>
        </property>
        <property name="title">
         <string>Connection Configuration</string>
        </property>
        <widget class="QLabel" name="label_12">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>30</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Select port :</string>
         </property>
        </widget>
        <widget class="QComboBox" name="typeBox_port">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>30</y>
           <width>471</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_connect">
         <property name="geometry">
          <rect>
           <x>680</x>
           <y>30</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Connect</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_13">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>80</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Console</string>
         </property>
        </widget>
        <widget class="QTextEdit" name="desc">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>100</y>
           <width>651</width>
           <height>441</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_resetPort">
         <property name="geometry">
          <rect>
           <x>590</x>
           <y>30</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Reset ports</string>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>649</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionOpen_csv_file"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_csv_file">
   <property name="text">
    <string>Open csv file</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionRecord">
   <property name="text">
    <string>Record</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header location="global">pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
